packages/angular-workspace/projects/angular/package.json

```diff
@@ -13,6 +13,6 @@
   },
   "dependencies": {
     "@astrouxds/astro-web-components": "^7.3.0",
-    "tslib": "^2.3.0"
+    "tslib": "^2.4.1"
```

Comment:
If the default angular 12 freshie install is tslib@2.3.0, is this going to cause issues for someone when they go to pull in astrouxds/angular?

Reply:
No and yes.

The default installation of this package can install any version of `tslib`, currently ranging from v2.3.0 to v2.4.1. This can depend on their machine, existing caching, the semver of other packages they are using, and their package manager. The `^` range explicitly allows any of these versions to be acceptable. The tooling reconciles these things for a user.